File: data/src/main/java/com/hgianastasio/biblioulbrav2/data/models/user/renew/RenewLoansResponseEntity.java
package com.hgianastasio.biblioulbrav2.data.models.user.renew;


import com.google.gson.annotations.SerializedName;


/**
 * Created by heitorgianastasio on 4/25/17.
 */
public class RenewLoansResponseEntity {
    @SerializedName("renovado")
    private String status;

    @SerializedName("razao")
    private String message;

    public RenewLoansResponseEntity() {
    }

    public RenewLoansResponseEntity(String status, String message) {
        this.status = status;
        this.message = message;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: data/src/main/java/com/hgianastasio/biblioulbrav2/data/models/user/renew/RenewLoansResponseEntity.kt
package com.hgianastasio.biblioulbrav2.data.models.user.renew

import com.google.gson.annotations.SerializedName

/**
 * Created by heitorgianastasio on 4/25/17.
 */
class RenewLoansResponseEntity {
    @SerializedName("renovado")
    var status: String? = null

    @SerializedName("razao")
    var message: String? = null

    constructor() {}
    constructor(status: String?, message: String?) {
        this.status = status
        this.message = message
    }

}